theano/tensor/nnet/tests/test_conv.py

```diff
@@ -281,8 +281,6 @@ def test_unroll_batch_kern_fail(self):
     @attr('slow')
     def test_subsample(self):
         # Tests convolution where subsampling != (1,1)
-
-        self.validate((3, 2, 7, 5), (5, 2, 2, 3), 'valid', subsample=(2, 2))
```

Comment:
This test fails after some changes to the master branch.
This is changed to
`self.assertRaises(NotImplementedError, self.validate, (3, 2, 7, 5), (5, 2, 2, 3), 'valid', subsample=(2, 2))`
which is in the code...
Is this what you mean?